src/Chapter3.hs

```diff
@@ -459,6 +482,7 @@ and provide more flexibility when working with data types.
 Create a simple enumeration for the meal types (e.g. breakfast). The one who
 comes up with the most number of names wins the challenge. Use your creativity!
 -}
+data Meal = Breakfast | Frühstück | Brunch | Lunch | Dinner | Supper | Snack | OnlyWater
```

Comment:
🧀 🍨 🧊 
These are new! Would need to add `Frühstück` into my schedule 😄

Reply:
It's the second German breakfast :partying_face: